engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -446,7 +409,7 @@ public CompilationStage getCompilationStage() {
    *
    * @param compilationStage the new compilation stage for the module.
    */
-  public void unsafeSetCompilationStage(CompilationStage compilationStage) {
+  void unsafeSetCompilationStage(CompilationStage compilationStage) {
```

Comment:
I wanted to do this change since I joined Enso. Calling a method `unsafe`, but exposing it to everyone is a poor design practice. Let's hide it!

The only way to call these `unsafe` method is to:
- be in the same package
- have access to `CompilerContext.updateModule`
- use `ModuleTestUtils` in tests

The access control shall make it clear it is not a method for everyone.